stable/datadog/templates/container-process-agent.yaml

```diff
@@ -0,0 +1,29 @@
+{{- define "container-process-agent" -}}
+- name: {{ .Chart.Name }}-process-agent
+  image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+  imagePullPolicy: {{ .Values.image.pullPolicy }}
+  command: ["process-agent", "-config=/etc/datadog-agent/datadog.yaml"]
+  resources:
+{{ toYaml .Values.daemonset.containers.processAgent.resources | indent 4 }}
+  env:
+    {{- include "containers-common" . | nindent 4 }}
+    - name: DD_PROCESS_AGENT_ENABLED
+      value: {{ .Values.datadog.processAgentEnabled | quote }}
+{{- if .Values.daemonset.containers.processAgent.env }}
+{{ toYaml .Values.daemonset.containers.processAgent.env | indent 4 }}
+{{- end }}
+  volumeMounts:
```

Comment:
don't we need /etc/passwd in the process agent?

Reply:
Yes otherwise username aren't resolved, good catch 👍